index.js

```diff
@@ -1,16 +1,24 @@
 /* jshint node: true */
 'use strict';
-
 var JSCSFilter = require('broccoli-jscs');
 var jscsrcBuilder = require('./lib/jscsrc-builder');
 
 module.exports = {
   name: 'ember-suave',
 
-  lintTree: function(type, tree) {
+  lintTree: function (type, tree) {
     var jscsOptions = this.app.options.jscsOptions || {};
     jscsOptions.configPath = jscsrcBuilder(this.project);
-
     return new JSCSFilter(tree, jscsOptions);
+  },
+
+  included: function(app, parentAddon) {
+    var target = (parentAddon || app);
+    this._super.included(target);
```

Comment:
I think this needs to call 

    this._super.included.apply(this, arguments);

(the super.included call needs to be bound to this)

---

Also, do we need the massaging of arguments to determine target like this or can we just call super with the args we were called with?